### PR TITLE
[PR] Fix tryptic

### LINKED ISSUE #141: [BUG]Add relation from tryptic doesn't work
**Description of the bug**
Add relation from / to (from the tryptic) doesn't work properly.
It correctly redirects to the add relation section, correctly fills the person but you can't save the relation -> as if the person isn't selected.

Ø

### LINKED ISSUE #142: [BUG]Tryptic doesn't display relations
**Description of the bug**
The tryptic doesn't display the relation associated to the person.

**Severity of the bug**

- It's annoying every time but you live with it

**Appearance of the bug**
How this bug happened :
1.Add a relation
2. Try to see the tryptic
3. BUG

**Idea**

Maybe the bug is due to the new id system
Ø

### PR DESCRIPTION
**Pull request by mathisfchx**
### Description
Fix tryptic

### CODE REVIEW

File: Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonDetailsRelationToAdapter.java [hunk 1]
```diff
@@ -26,8 +27,8 @@ public class PersonDetailsRelationToAdapter  extends ArrayAdapter<Relation> {
      * A basic viewholder to put a person's name and a relation type
      */
     private static class ViewHolder {
-        TextView name_person;
-        TextView relation_type;
+        TextView bdv;
```
Comment (lduf) [inline Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonDetailsRelationToAdapter.java:30]: This file is almost the same as PersonDetailsRelationFromAdapter, is that possible to merge them ?

**Review by lduf — APPROVED** [review]
🌍

File: Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonDetailsRelationToAdapter.java (hunk 1 above)
Comment (mathisfchx) [reply Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonDetailsRelationToAdapter.java:30]: maybe with reflectivity ? #134

Comment (lduf) [reply Netw4ppl/app/src/main/java/netw4ppl/ines/utils/PersonDetailsRelationToAdapter.java:30]: guess so

**Pull request merged**